Source/OrchBase/Composer.cpp

```diff
@@ -1188,24 +1840,23 @@ void Composer::writeMessageTypes(GraphI_t* graphI, std::ofstream& pkt_h)
 
     pkt_h << "#include <cstdint>\n";
     //pkt_h << "#include \"softswitch_common.h\"\n\n";
```

Comment:
More comments, which may or may not be useful?

Reply:
That comment was an old one, the pragmas are there as a reminder to investigate something further.